src/posts/2020-04-16-continuous-stuff-with-github-actions/index.md

```diff
@@ -0,0 +1,262 @@
+---
+id: 147
+path: /posts/continuous-stuff-with-github-actions
+title: "Continuous Stuff with Github Actions"
+author: miguel-palhas
+date: 2020-04-20
+tags:
+  - devops
+intro: "Last year, I took on the task of improving the continuous process over at Utrust. We weren't really happy with the amount of work that went into our releases, and I was looking for a more agile approach, where everyone from developers to the QA team could do their part with low friction."
+---
+
+[slides]: https://slides.com/naps62/continuous-stuff
+[utrust]: https://utrust.com
+[github-actions]: https://github.com/features/actions
+[circleci]: http://circleci.com/
+[primer]: https://www.imdb.com/title/tt0390384/
+[github-deployments]: https://developer.github.com/v3/repos/deployments/
+[github-deployment-status]: https://developer.github.com/v3/repos/deployments/
+[github-workflow-triggers]: https://help.github.com/en/actions/reference/events-that-trigger-workflows
+[circleci-orbs]: https://circleci.com/orbs/
+[orb-awscli]: https://circleci.com/orbs/registry/orb/circleci/aws-cli#orb-source
+[actions-checkout]: https://github.com/actions/checkout/tree/v2.0.0
+[twitter]: https://twitter.com/naps62
+
+<div class="custom-paragraph">
+<em>
+This post was adapted from a talk that I gave at several local meetups over the past few months. It got enough positive
+feedback that I felt compeled to convert it to written format. Original slides are available <a
+href="https://slides.com/naps62/continuous-stuff" target="_blanks">here</a>.
```

Comment:
You don't need to use literal anchor tags to get the blank target, our posts will automatically render links like this (thanks to `gatsby-remark-external-links`, added in #286).

Also just noticed a typo: compel**l**ed :grimacing: 

```suggestion
feedback that I felt compelled to convert it to written format. Original slides are available [here][slides].
```

Reply:
the literal anchor tag is because I can't use markdown inside html tags